README.md

```diff
@@ -608,6 +610,40 @@ The DLP templates used in this tutorial are specifically tailored for inspecting
 2. [Create de-identification templates and run de-identification on sample data](https://cloud.google.com/dlp/docs/creating-templates-deid)
 
 
+## Run this pipeline outside Google Cloud shell
+
+Google Cloud Shell comes with a pre-configured development environment that includes many of the most popular tools and
+libraries, such as gcloud, gradle, java, git, vim, and more. This means that you don't have to spend time setting up 
+your own environment on your local machine. If you want to run this solution from your own machine, follow the 
+instructions given below.
+
+1. Create a new GCP project.
+2. Set up the tools and libraries required to run the DLP Dataflow pipelines including gradle (recommended version: 8.3),
```

Comment:
RE: Set up the tools and libraries required to run the DLP Dataflow pipelines

What are the tools and libraries required? It would be good to have a section that lists the required tools. That section is a good place for the recommended versions, so you don't have to write them inline here.